pkg/controller/tenant/tenant_reconcile.go

```diff
@@ -32,5 +32,13 @@ func (r *ReconcileTenant) Reconcile(request reconcile.Request) (reconcile.Result
 		FinalizerName: "",
 	}
 
-	return reconcile.Result{}, pipeline.ReconcileTenant(instance, data)
+	err = pipeline.ReconcileTenant(instance, data)
+	if err != nil {
+		if errors.IsConflict(err) {
```

Comment:
Does this still work, even though we wrap the error in `pipelines.go:56`?
`fmt.Errorf("step %s failed: %w", step.Name, r.Err)`

Reply:
It might need a call to `Unwrap`: `apierror.IsConflict(errors.Unwrap(err))`

Reply:
Yeah, no... I changed it.